### PR TITLE
hw-mgmt: patches: Align patch table

### PR DESCRIPTION
**Pull request by vadimp-nvidia**
Align patch table to match Sonic canonical build.

### CODE REVIEW

**Comment by ciju-nvidia** [comment]
RERUN

File: recipes-kernel/linux/Patch_Status_Table.txt [hunk 1]
```diff
@@ -525,7 +536,7 @@ Kernel-6.1
 |0035-i2c-mlxcpld-Support-PCIe-mapped-register-space.patch        |                    | Feature pending                          |            |                                                |
 |0036-mlxsw-i2c-Fix-chunk-size-setting-in-output-mailbox-b.patch  |                    | Bugfix pending                           |            |                                                |
 |0037-mlxsw-i2c-Limit-single-transaction-buffer-size.patch        |                    | Bugfix pending                           |            |                                                |
-|0038-mlxsw-core_hwmon-Adjust-module-label-names-based-on-.patch  |                    | Feature pending                          |            |                                                |
+|0038-mlxsw-core_hwmon-Adjust-module-label-names-based-on-.patch  |                    | Downsteam                                |            |                                                |
```
Comment (sholeksandr) [inline recipes-kernel/linux/Patch_Status_Table.txt:539]: Misprint:

Downsteam => Downstream

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by MellanoxBSP** [comment]
RERUN

**Review by sholeksandr — APPROVED** [review]

**Comment by MellanoxBSP** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

**Pull request merged**